deploy.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
I still wonder if it makes more sense to just checkout the repo and start the docker containers from there like we do in ci.yml instead of all the docker load/save, cp docker-compose.yml shenanigans... 
It's ok for now but maybe we can have a chat about it some time.

Reply:
Probably, yeah... When we started out with the project I didn't expect the infrastructure needed to host a django app to be this ridiculously complex.